### ISSUE TITLE
Tarantool 1.9 support

### ISSUE THREAD

**Issue by GhOsTMZ** [tarantool/memcached#44]
Hi,
I was install a memcached module and have this at startup tarantool:
```2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> LuajitError at /build/usr/src/debug/tarantool-1.9.0.87/src/lua/utils.c:923
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua F> ./memcached.lua:49: loop or previous error loading module 'memcached'
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> tuple_delete(0x15b2500)
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> vy_tuple_delete(0x15b2500)
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> cpipe_flush_cb: locking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> cpipe_flush_cb: unlocking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] wal/101/main D> cbus_endpoint_fetch: locking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] wal/101/main D> cbus_endpoint_fetch: unlocking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] wal/101/main D> cpipe_destroy: locking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] wal/101/main D> cpipe_destroy: unlocking &endpoint->mutex
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> cbus_destroy: locking &bus->mutex
2018-05-07 14:17:46.884 [10871] main/101/memcached.lua D> cbus_destroy: unlocking &bus->mutex
```

This module have a support Tarantool 1.9?

Thanks!

**Comment by GhOsTMZ** [comment]
I was rebuild memcached module with Tarantool 1.9 headers and have this at startup (used builded for 1.7 before):
```
2018-10-08 12:32:06.839 [10465] main/101/memcached expiration.c:114 E> Can't start the expire fiber                                                                                                                
2018-10-08 12:32:06.839 [10465] main/101/memcached expiration.c:115 E> fio: No such file or directory                                                                                                              
2018-10-08 12:32:06.839 [10465] main/101/memcached expiration.c:114 E> Can't start the expire fiber                                                                                                                
2018-10-08 12:32:06.839 [10465] main/101/memcached expiration.c:115 E> fio: No such file or directory
```

**Comment by GhOsTMZ** [comment]
Fixed in #46
